workflow/scripts/create_somalier_mqc_config.py

```
@@ -400,8 +486,25 @@ def get_expected_relatedness(sample_a, sample_b, ped_df):
+    parser.add_argument('--ped', required=True, help='PED file input')
+    parser.add_argument('--config', required=True, help='MultiQC config YAML input')
+    parser.add_argument('--rel-check-mqc', required=True, help='Output relatedness TSV')
+    parser.add_argument('--sex-check-mqc', required=True, help='Output sex check TSV')
+    parser.add_argument('--general-stats-mqc', required=True, help='Output general stats TSV')
+    return parser.parse_args()
+
+
 if __name__ == "__main__":
     try:
+        # Check if running via Snakemake script directive
+        snakemake  # noqa: F821 - injected by Snakemake
```

Ruff (0.15.14)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)